src/platform-includes/enriching-events/breadcrumbs/automatic-breadcrumbs/react-native.mdx

```diff
@@ -0,0 +1,12 @@
+The React Native SDK already captures breadcrumbs automatically via the Native SDKs for Android and iOS.
+
+- [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+- [Automatic Breadcrumbs for iOS](/platforms/apple/configuration/integrations/default/#sentryautobreadcrumbtrackingintegration)
+
+- For interaction breadcrumbs wrapp your `App` using the [`Sentry.wrap`](/platforms/react-native/performance/instrumentation/automatic-instrumentation/#wrap-your-root-component) method.
+
+- For navigation breadcrumbs use one of the [SDK's natigation integrations](/platforms/react-native/performance/instrumentation/automatic-instrumentation/#enable-routing-instrumentation).
+
+- `HTTP` requests are included in breadcrumbs by default.
+
+- To track redux actions and latest store state use [`Sentry.createReduxEnhancer`](/platforms/javascript/guides/react/configuration/integrations/redux).
```

Comment:
The two items above are clearly related to the sentence that comes before them, but these four items aren't and shouldn't be part of the same list as those first two items. My recommendation would be to just make this a paragraph. If you want to keep these four items as a separate bullet list, then an introductory sentence needs to be added that makes clear why they're all in the same list. 

Also, is `App` here a setting or property or do you just mean the application?
 
```suggestion
For interaction breadcrumbs, wrap your `App` using the [`Sentry.wrap`](/platforms/react-native/performance/instrumentation/automatic-instrumentation/#wrap-your-root-component) method. For navigation breadcrumbs, use one of the [SDK's navigation integrations](/platforms/react-native/performance/instrumentation/automatic-instrumentation/#enable-routing-instrumentation). `HTTP` requests are included in breadcrumbs by default. To track redux actions and the latest store state, use [`Sentry.createReduxEnhancer`](/platforms/javascript/guides/react/configuration/integrations/redux).
```

Reply:
Thanks for the comment.

By `App` I meant a root component. I've changed that. 

For the rest, I would like to keep it as a list, so I've added the introductory sentence.